schemas/jsonschema/CameraCalibration.json

```diff
@@ -1,6 +1,5 @@
 {
-  "$comment": "Generated from CameraCalibration by @foxglove/schemas",
-  "title": "CameraCalibration",
+  "title": "foxglove.CameraCalibration",
```

Comment:
This seems more correct as a title since the `foxglove.` namespace part is important.

Removing the comment on where these were generated is a downgrade.

Reply:
Restored comment